secret-provider/src/test/scala/io/lenses/connect/secrets/providers/VaultSecretProviderTest.scala

```diff
@@ -324,6 +324,27 @@ class VaultSecretProviderTest extends AnyWordSpec with Matchers with BeforeAndAf
     response.getData.asScala("value") shouldBe "mock"
   }
 
+  "should be configured for vault engine prefix depth" in {
```

Comment:
there is a test for explicitly setting the prefix depth, can there also be a test to ensure the default value is 1?

Reply:
I also added a test for the default 👍

Reply:
@davidsloan Does this look good to you?